src/librustc/ty/context.rs

```diff
@@ -1307,10 +1307,6 @@ impl<'tcx> TyCtxt<'tcx> {
         self.all_crate_nums(LOCAL_CRATE)
     }
 
-    pub fn injected_panic_runtime(self) -> Option<CrateNum> {
-        self.cstore.injected_panic_runtime()
-    }
```

Comment:
Miri was relying on this method... is there a replacement?

Reply:
Hmm, something like
```rust
tcx.crates().iter().find(|cnum| tcx.is_panic_runtime(cnum))
```
should work without re-exposing things privatized in this PR.

(If that's not ok, I can send a PR re-adding `tcx.injected_panic_runtime()`.)

Reply:
That seems to work, thanks! (modulo some extra `*`)

See https://github.com/rust-lang/miri/pull/1085